docs/sphinx/contribution/code-style-python.rst

```diff
@@ -404,4 +404,34 @@ At this time, Assignment Expressions (Walrus operator) are not supported.
      - Assignment Expressions
      - Adds a way to assign to variables within an expression
      - 3.8
-     - No
\ No newline at end of file
+     - No
+
+
+Deprecate distutils module support
+==================================
+
+Python 3.12 will deprecate and remove the distutils module.  In order for CHIPSEC to support these versions of Python, setuptools should be used instead of distutils.
```

Comment:
```suggestion
Python 3.12 will deprecate and remove the distutils module.  In order for CHIPSEC to support this version of Python, setuptools should be used instead of distutils.
```